.github/workflows/build.yml

```diff
@@ -29,14 +29,10 @@ jobs:
           # Ubuntu
           - focal_amd64
           - bionic_amd64
-          - xenial_amd64
           # RedHat family
           - el9_x86_64
           - el8_x86_64
```

Comment:
It seems we missed CentOS 8 EOL 
```CentOS Linux 8 will reach End Of Life (EOL) on December 31st, 2021. ```, see 
https://www.centos.org/centos-linux-eol/

Reply:
I didn't realize the "EL" builders were actually using CentOS. With CentOS being discontinued, what would be the best way forward here? Are there free to use variants of RHEL that we could test instead?

Reply:
I would start with CentOS stream (that's basically current RHEL). Or we could use almalinux here?
https://hub.docker.com/_/almalinux